packages/@holaplexui-playground/pages/form.tsx

```diff
@@ -129,15 +131,17 @@ export default function App() {
         </Form.Select>
 
         <Form.Select
-          className="mt-5"
+          className='mt-5'
           multiple
           onChange={(v) => {
             setMultiValue(v);
           }}
           value={multiValue}
         >
-          <Form.Select.Button icon={<Icon.Sol />} placeholder="e.g. Solana">
-            {singleValue?.name}
+          <Form.Select.Button icon={<Icon.Sol />} placeholder='e.g. Solana'>
+            {multiValue && multiValue?.length > 0
+              ? multiValue[0].name
+              : undefined}
```

Comment:
Lets fix the children prop of this if its undefined then show placeholder.

Reply:
Yes, it already shows the placeholder if children are undefined.